docs/chapters/architecture/SOTA-system.rst

```diff
@@ -0,0 +1,614 @@
+| Requirement ID | Requirement description                                     |
++================+=============================================================+
+| RA-1           | Atomic updates                                              |
++----------------+-------------------------------------------------------------+
+| RA-2           | Automatic revert when a non-working update is applied       |
++----------------+-------------------------------------------------------------+
+| RA-3           | Updates integrity and signature verification                |
++----------------+-------------------------------------------------------------+
+| RA-4           | Fleet management (controlled deployment and monitoring)     |
++----------------+-------------------------------------------------------------+
+
+More information on requirements in the :ref:`arch-vertical-configurations`.
+
+Quality goals
+^^^^^^^^^^^^^
+
++----------+--------------+----------------------------------------------------+
+| Priority | Quality-Goal | Scenario                                           |
++==========+==============+====================================================+
+| 1        | Safety       | A PELUX system should never end up in a corrupted  |
+|          |              | or inconsistent state as a result of a SOTA system |
+|          |              | operation.                                         |
++----------+--------------+----------------------------------------------------+
+| 1        | Security     | If configured accordingly, any update artifact     |
+|          |              | which has not been signed by a PELUX Release       |
+|          |              | manager should not be applied.                     |
++----------+--------------+----------------------------------------------------+
+| 2        | Correctness  | The result of an update should be exactly the one  |
+|          |              | intended by the Release manager.                   |
++----------+--------------+----------------------------------------------------+
+| 3        | Flexibility  | PELUX release managers should be kept informed of  |
+|          |              | all update failures and able to cancel an update   |
+|          |              | deployment.                                        |
++----------+--------------+----------------------------------------------------+
+| 4        | Usability    | Update artifacts should be distributed in a        |
+|          |              | compressed format to minimize the time needed for  |
+|          |              | an update and the network usage.                   |
++----------+--------------+----------------------------------------------------+
+
+Stakeholders
+^^^^^^^^^^^^
+
++-----------------------+--------------------------+---------------------------+
+| Role                  | Description              | Goal, Intention           |
```

Comment:
What about the user of the SOTA software in the car?

Reply:
I am not sure how/if the user is supposed to interact with the SOTA system. This sounds like a UX problem to me.
Some GUI implementations could decide to apply the updates by themselves or let the user decide if he wants to apply updates later. In each case, I would argue that the user only interacts with the GUI and the GUI interacts with the SOTA system itself as described in the first row of this table.

But maybe I am misunderstanding! :)